### PR TITLE
Won lost pages and reset game

### PR DESCRIPTION
**Pull request by fabjab86**
Merging with other branches.

### CODE REVIEW

File: Femo1/QuestionScene/QuestionScene.swift [hunk 1]
```diff
@@ -57,6 +57,7 @@ class QuestionScene: SKScene {
         
         if isKeyPresentInUserDefaults(key: "questionOrderArray") {
             questionOrderArray = userDefaults.array(forKey: "questionOrderArray") as! [Int]
+            print("from defaults: \(questionOrderArray)")
```
Comment (Mschmidt19) [inline Femo1/QuestionScene/QuestionScene.swift:60]: will want to delete this

File: Femo1/QuestionScene/QuestionScene.swift [hunk 2]
```diff
@@ -100,8 +103,23 @@ class QuestionScene: SKScene {
         for i in 0...questionList.count-1 {
             tempArray.append(i)
         }
-        tempArray.shuffle()
-        questionOrderArray = tempArray
+        questionOrderArray = shuffleArray(array: tempArray)
+        print("from create: \(questionOrderArray)")
```
Comment (Mschmidt19) [inline Femo1/QuestionScene/QuestionScene.swift:107]: and delete this

**Review by Eviekim — APPROVED** [review]
Well done 💃

**Review by Eviekim — APPROVED** [review]

**Review by Mschmidt19 — APPROVED** [review]
Looks great! Nice shuffle function for xCode 9 compatibility. Just need to clean up a few remaining print statements

**Review by Eviekim — APPROVED** [review]

**Review by Eviekim — APPROVED** [review]

**Pull request merged**